ui.apps/src/main/content/jcr_root/apps/core/fd/components/aemform/v1/aemform/iframe.html

```diff
@@ -20,6 +20,7 @@
          data-sly-use.templates="core/wcm/components/commons/v1/templates.html"/>
     <title data-sly-test="${form.isAdaptiveForm || form.isMCDocument}"><sly data-sly-use.af="com.adobe.aemds.guide.common.AdaptiveForm" />${af.formTitle @ context='html'}</title>
     <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta name="robots" content="noindex, nofollow"/>
```

Comment:
Can we add an e2e test case for this (ie) when iframe mode is selected, we validate the markup to contain this ?

Reply:
okay
Will add an e2e case for this alongside e2e tests for link component